apps/typegpu-docs/src/content/examples/rendering/cubemap-reflection/index.ts

```diff
@@ -403,13 +403,13 @@ canvas.addEventListener('touchstart', (event) => {
     prevX = event.touches[0].clientX;
     prevY = event.touches[0].clientY;
   }
-});
+}, { passive: true });
 
-window.addEventListener('mouseup', () => {
+canvas.addEventListener('mouseup', () => {
```

Comment:
Attaching these events to the canvas breaks when dragging outside of the canvas and letting go.